README.md

```diff
@@ -10,53 +10,72 @@
-   possible, `relude` tries to depend only on boot libraries.
-   [Dependency graph](relude-dependency-graph.png) of `relude` can give you clearer picture.
-5. **Convenience** (e.g lifted to `MonadIO` functions, more reexports). Despite minimalism, we
-   want to bring common types and functions (like `containers` and `bytestring`)
-   into scope because they are used in almost every application anyways.
-6. **Provide excellent documentation.**
+5. **Minimalism** (low number of dependencies). We don not force users of
+   `relude` to stick to any specific lens or text formatting or logging
+   library. Where possible, `relude` depends only on boot libraries.
+   The [Dependency graph](https://raw.githubusercontent.com/kowainik/relude/master/relude-dependency-graph.png)
+   of `relude` can give you a clearer picture.
+6. **Convenience** (e.g. lifted to `MonadIO` functions, more reexports). Despite
+   minimalism, we want to bring common types and functions (like `containers`
+   and `bytestring`) into scope because they are used in almost every
+   application anyways.
+7. **Provide excellent documentation.**
    + Tutorial
    + Migration guide from `Prelude`
-   + Haddock for every function with examples tested by [`doctest`](http://hackage.haskell.org/package/doctest)
+   + Haddock for every function with examples tested by
+     [`doctest`](http://hackage.haskell.org/package/doctest)
    + Documentation on [internal module structure](http://hackage.haskell.org/package/relude/docs/Relude.html)
-   + `relude`-specific [HLint](http://hackage.haskell.org/package/hlint) rules: [`.hlint.yaml`](.hlint.yaml)
-7. **User-friendliness.** Ability to quickly migrate to `relude` if you're familiar
-   with the common libraries like `text` and `containers`.
-8. **Exploration.** Experiment with new ideas and proposals without introducing
-   breaking changes. `relude` uses the approach with `Extra.*` modules which are not
-   exported by default so it's quite easy to bring something new and let users
-   decide to use it or not.
-
-This README contains introduction to `relude` and a tutorial on how to use it.
-
-For an introduction to the alternative preludes, check the following
+   + `relude`-specific [HLint](http://hackage.haskell.org/package/hlint) rules:
+     [`.hlint.yaml`](https://github.com/kowainik/relude/blob/master/.hlint.yaml)
+8. **User-friendliness.** Anyone is able to quickly migrate to `relude`. Only
+   some basic familiarity with the common libraries like `text` and `containers`
+   should be helpful (but not necessary).
+9. **Exploration.** We have space to experiment with new ideas and proposals
+   without introducing breaking changes. `relude` uses the approach with
+   `Extra.*` modules which are not exported by default, so it makes it quite
+   easy for us to provide new functionality without breaking anything and let
+   the users decide to use it or not.
+
+<!-- TODO: anti-goals? -->
```

Comment:
Yes, I think it will be an excellent addition to the README to tell our anti-goals as well 👍